playwright.config.ts

```diff
@@ -34,12 +34,20 @@ export default defineConfig({
   projects: [
     {
       name: "chromium",
-      use: { ...devices["Desktop Chrome"] },
+      use: { ...devices["Desktop Chrome"], permissions: ["clipboard-read", "clipboard-write"] },
```

Comment:
I added these settings in the Playwright configuration to allow access to the clipboard. Thanks to this article, https://playwrightsolutions.com/how-do-i-access-the-browser-clipboard-with-playwright/ and this GitHub comment, https://github.com/microsoft/playwright/issues/13037#issuecomment-1667307993.